src/ios/THEPeerBluetooth.h

```diff
@@ -36,7 +36,7 @@
 
```

Comment:
FIX AFTER CHECKIN (if ever) - On line 35 a property is defined to hold the delegate. But wouldn't it be cleaner and safer to just submit the delegate as part of the constructor and then have an accessor when it is needed?